components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/store/SessionDataStore.java

```diff
@@ -39,13 +40,12 @@
     public static final BlockingDeque<SessionContextDO> sessionContextQueue = new LinkedBlockingDeque();
     private static final String CACHE_MANAGER_NAME = "IdentityApplicationManagementCacheManager";
     private static final Log log = LogFactory.getLog(SessionDataStore.class);
-    public static boolean redisEnabled = false;
     private static int maxSessionDataPoolSize = 100;
     private static boolean enablePersist;
     private static volatile SessionDataStore instance;
+    private static  SessionDataStore DEFAULT_SESSION_DATA_STORE = new RdbmsMultiEntrySessionDataStore();
```

Comment:
```suggestion
    private static  SessionDataStore DEFAULT_SESSION_DATA_STORE = new RdbmsMultiEntrySessionDataStore();
```
Let's not instantiate here. Set the store name as the value . Ex: Refer to the constant which stands for RdbmsMultiEntrySessionDataStore's name(one that get returned with the get name method).